text/0012-frameworks.md

```diff
@@ -0,0 +1,52 @@
+# Frameworks at Canopy
+- RFC PR: https://github.com/CanopyTax/frontend-rfc/pull/12
+
+## Definition
+There are a lot of front-end frameworks out there and we use several at Canopy. By design we've sought to avoid locking ourselves
+into a specific framework and becoming an `${frameworkX} shop`. This has come with a lot of pros and cons. Our goal is for the 
+best tools to "bubble up", rather than be selected and pushed "top down".
+
+## Why can we be so flexible with frameworks
+- [single-spa](https://github.com/CanopyTax/single-spa)
+- [SystemJS](https://github.com/systemjs/systemjs)
+- [SOFE](https://github.com/CanopyTax/sofe)
+
+### pros
+- Easy to build, experiment, and change our tech stack.
+- We aren't locked into Angular or React.
+- We can, theorectially, choose the best framework for the job.
+
+### cons
```

Comment:
Another con is a potentially lesser ability to transfer devs between squads

Reply:
Another con is initial load time performance -- more framework code means more kbs